docs/src/python/user-guide/expressions/user-defined-functions.py

```diff
@@ -11,22 +11,25 @@
         "values": [10, 7, 1],
     }
 )
+print(df)
+# --8<-- [end:dataframe]
 
+# --8<-- [start:shift_map_batches]
 out = df.group_by("keys", maintain_order=True).agg(
-    pl.col("values").map_batches(lambda s: s.shift()).alias("shift_map"),
+    pl.col("values").map_batches(lambda s: s.shift()).alias("shift_map_batches"),
```

Comment:
Shouldn't this be `map_elements`? A shift here would be incorrect? (Don't read the context).

Reply:
Right! Reviewing lost snippets is hard. :sweat: